model/model.go

```diff
@@ -0,0 +1,19 @@
+package model
+
+//User struct
+type User struct {
+	Username  string `json:"username"`
+	FirstName string `json:"firstname"`
+	LastName  string `json:"lastname"`
+	Password  string `json:"password"`
+	Gender    string `json:"gender"`
+	Country   string `json:"country"`
+	Age       string `json:"age"`
+	Email     string `json:"email"`
+}
+
+//ResponseResult struct
+type ResponseResult struct {

```

Comment:
Structs like User are used by handlers package and db package both, that's why we keep them in separate package called model. But ResponseResult struct is specific to handlers package and will not be used elsewhere, so keep  it in handlers package only.

Reply:
okay.